components/brave_rewards/extension/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "1.0.0",
+  "name": "Brave Rewards",
+  "manifest_version": 2,
+  "description": "Brave Rewards Extension",
+  "browser_action": {
+    "default_popup": "extension/popup.html",
+    "default_icon": "extension/img/BAT_icon_placeholder_32.png"
+  },
```

Comment:
extensions should have CSP defined, tho probably fine since it's still just a stub right now.

Reply:
Right, my assumption is that whomever implements the actual extension will add the appropriate CSP.